### PR TITLE
Search App added

### PR DESCRIPTION
**Pull request by madhuparnab**

### CODE REVIEW

**Review by sidhantgoyal-droid — COMMENTED** [review]
Lets rename the directory with proper casing as "demo_search_app"

**Comment by madhuparnab** [comment]
> Lets rename the directory with proper casing as "demo_search_app"

Done.

File: demo_search_app/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Files
```
Comment (sidhantgoyal-droid) [inline demo_search_app/README.md]: Maybe delete this

**Review by sidhantgoyal-droid — DISMISSED** [review]

**Review by ashish-google — DISMISSED** [review]

File: demo_search_app/README.md (hunk 1 above)
Comment (madhuparnab) [reply demo_search_app/README.md]: done

**Review by sidhantgoyal-droid — APPROVED** [review]

**Review by ashish-google — APPROVED** [review]

**Pull request merged**